templates/sheets/alchemical-sheet.html

```diff
@@ -1,19 +1,22 @@
 <form class="{{cssClass}}" autocomplete="off">
     <header class="sheet-header">
         <img class="profile-img" src="{{item.img}}" data-edit="img" title="{{item.name}}" />
-        <div class="header-fields">
-            <h1 class="charname"><input name="name" type="text" value="{{item.name}}" placeholder="Name" /></h1>
-            <div class="resource flex">
-                <div class="flex">
-                    <label>{{localize "WITCHER.Item.Quantity"}}</label>
-                    <input type="text" name="data.quantity" value="{{data.quantity}}" data-dtype="Number" />
-                    <label>{{localize "WITCHER.Item.Weight"}}</label>
-                    <input type="text" name="data.weight" value="{{data.weight}}" data-dtype="Number" />
-                    <label>{{localize "WITCHER.Item.Cost"}}</label>
-                    <input type="text" name="data.cost" value="{{data.cost}}" data-dtype="Number"/>
-                </div>
-            </div>
+        <h1 class="charname"><input name="name" type="text" value="{{item.name}}" placeholder="Name" /></h1>
+    </header>
+    <div class="item-row">
```

Comment:
Almost every item sheet is displaying the Quantity, Weight and Cost at the same place. 
Maybe we should create a template and use it in the sheets

Reply:
It's a good Idea, Will also add name and picture to that template. I have a design ide I will try out.